stage0/run.go

```diff
@@ -314,7 +314,7 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 
 // Run mounts the right overlay filesystems and actually runs the prepared
 // pod by exec()ing the stage1 init inside the pod filesystem.
-func Run(cfg RunConfig, dir string) {
+func Run(cfg RunConfig, dir string, dataDir string) {
```

Comment:
I was wondering if we should keep lkvm-flavor specific stuff in different directory then rkt's datadir (usually `/var/lib/rkt`). For example in `/var/lib/rkt-kvm` (or `/var/lib/coreos/rkt/stage1/lkvm` or something else). That would allow us to move SSH setup code from stage0 to stage1, where it should be IMO.